rtnetlink/examples/ip_monitor.rs

```diff
@@ -1,6 +1,6 @@
 use futures::stream::StreamExt;
 
-use netlink_proto::sys::constants::*;
+use netlink_packet_route::rtnl::constants::*;
```

Comment:
I'd prefer a shorter one.

```suggestion
use netlink_packet_route::constants::*;
```

Reply:
done, thanks